MODULE.bazel

```diff
@@ -0,0 +1,33 @@
+module(
+    name = "rules_jvm_external",
+    version = "1.0"
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.1.1")
+bazel_dep(name = "rules_java", version = "5.0.0")
```

Comment:
What are we using rules_java for? IIUC the Starlark rewrite uses "builtins" to manage migration now, so no need for rules_java loads.